music_assistant/controllers/media/albums.py

```diff
@@ -111,6 +111,7 @@ async def library_items(
         provider: str | list[str] | None = None,
         extra_query: str | None = None,
         extra_query_params: dict[str, Any] | None = None,
+        genre_items_ids: list[int] | None = None,
```

Comment:
I would just name this parameter genres and then explain in the docstring that you want library genre id's here